ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/textinput/v1/textinput/textinput.html

```diff
@@ -48,7 +48,8 @@
               placeholder="${text.placeHolder}"
               autocomplete="${text.autoComplete}"
               minlength="${text.minLength}"
-              maxlength="${text.maxLength}"></textarea>
+              maxlength="${text.maxLength}"
+              dir="auto"></textarea>
```

Comment:
Why is this change required ? Applying dir="auto" at the form level is generally sufficient when have consistent text directionality.

Reply:
On form container level, we have added dir=rtl (based on locale), but on every input field we have to add dir=auto. As input direction is dependent on the text user inserted inside it.